### DISCUSSION TITLE
Toggle for backpack.fullinventory.toggle true

### DISCUSSION THREAD

**Discussion by toetagger** [GeorgH93/Minepacks#293] [Ideas]
So in luckperms backpack.fullinventory.toggle seems to exist, however there appears to be no interaction with the user. 

Make it so when the backpack.fullinventory.toggle permisison is given that the user can toggle on or off the auto pickup. 

Thanks
Andy / Cpt.FishPants.

**Comment by GeorgH93** [comment]
You have to enable it in the config first. Please search for `IsToggleAllowed` and change that value to `true`.
